crates/net/network/src/transactions.rs

```diff
@@ -335,6 +335,56 @@ where
         propagated
     }
 
+    /// Propagate the transaction to single peer as hash
+    fn propagate_hash_to(&mut self, hash: TxHash, peer_id: PeerId) {
+        // Nothing to propagate while initially syncing
+        if self.network.is_initially_syncing() {
+            return
+        }
```

Comment:
we can skip this

Reply:
Done https://github.com/paradigmxyz/reth/pull/4713/commits/4ae046f34fb5b63bb6362691c1d8e73ff3089ee8